_posts/2016-09-22-ignobel.md

```diff
@@ -0,0 +1,50 @@
+---
+layout: post
+title: Want to win an Ig Nobel? Here are some suggestions.
+author: neuroamanda
+---
+
+<blockquote class="twitter-tweet" data-lang="en"><p lang="en" dir="ltr"><a href="https://twitter.com/matthewcobb">@matthewcobb</a> <a href="https://twitter.com/ProfBrianCox">@ProfBrianCox</a> got to be worth an Ignobel...??</p>&mdash; Russell Blinman (@Georgiasdad1) <a href="https://twitter.com/Georgiasdad1/status/771659917308547072">September 2, 2016</a></blockquote>
```

Comment:
no, not intentionally quoting an anit-EU person. i'm not sure the embeds are showing up correctly, because i also wanted it to display the parent tweet (i.e. a two-tweet conversation), it doesn't make sense without it.